build.gradle

```diff
@@ -112,6 +112,11 @@ tasks.named("publishMavenJavaPublicationToMavenLocal").configure { dependsOn("ge
 tasks.named("publishNebulaPublicationToMavenLocal").configure { dependsOn("generatePomFileForMavenJavaPublication") }
 validateMavenJavaPom.enabled = false
 
+javadoc {
+    source = sourceSets.main.allJava
+    classpath = sourceSets.main.compileClasspath
+}
```

Comment:
This was removed as a part of this [PR](https://github.com/opensearch-project/opensearch-sdk-java/pull/800/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7L97)